common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -193,8 +199,8 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
     }
 
     @Override
-    public boolean containsValue(V value) {
-        V v1 = toInternal(value);
+    public boolean containsValue(Object value) {
+        V v1 = toInternal((V) value);
```

Comment:
@nmittler - Aren't we casting without checking? Seems like this is an appropriate warning.